src/tests/ftest/container/open.py

```diff
@@ -0,0 +1,199 @@
+
+        # pool 2 UID GID
+        self.createuid2 = self.params.get("uid", '/run/createtests/createuid2/')
+        self.creategid2 = self.params.get("gid", '/run/createtests/creategid2/')
+
+        self.agent_sessions = AgentUtils.run_agent(self.basepath, self.hostlist)
+        server_utils.run_server(self.hostfile, self.server_group, self.basepath)
+
+    def tearDown(self):
+        try:
+            if self.container1 is not None:
+                self.container1.destroy()
+            if self.container2 is not None:
+                self.container2.destroy()
+            if self.pool1 is not None and self.pool1.attached:
+                self.pool1.destroy(1)
+            if self.pool2 is not None and self.pool2.attached:
+                self.pool2.destroy(1)
+        finally:
+            if self.agent_sessions:
+                AgentUtils.stop_agent(self.hostlist, self.agent_sessions)
+            server_utils.stop_server(hosts=self.hostlist)
+
+    def test_container_open(self):
+        """
+        Test basic container bad create.
+
+        :avocado: tags=container,containeropen
+        """
+        container_uuid = None
+        expected_for_param = []
+        uuidlist = self.params.get("uuid", '/run/createtests/uuids/*/')
+        container_uuid = uuidlist[0]
+        expected_for_param.append(uuidlist[1])
+
+        pohlist = self.params.get("poh", '/run/createtests/handles/*/')
+        poh = pohlist[0]
+        expected_for_param.append(pohlist[1])
+
+        expected_result = 'PASS'
+        for result in expected_for_param:
+            if result == 'FAIL':
+                expected_result = 'FAIL'
+                break
+
+        try:
+            # create two pools and try to create containers in these pools
+            self.pool1 = DaosPool(self.context)
+            self.pool1.create(self.createmode, self.createuid1, self.creategid1,
+                              self.createsize, self.createsetid, None)
+
+            self.pool2 = DaosPool(self.context)
+            self.pool2.create(self.createmode, self.createuid2, self.creategid2,
+                              self.createsize, None, None)
+
+            # Connect to the pools
+            self.pool1.connect(1 << 1)
+            self.pool2.connect(1 << 1)
+
+            # defines pool handle for container open
+            if pohlist[0] == 'pool1':
+                poh = self.pool1.handle
+            else:
+                poh = self.pool2.handle
+
+            # Create a container in pool1
+            self.container1 = DaosContainer(self.context)
+            self.container1.create(self.pool1.handle)
+
+            # defines test UUID for container open
+            if uuidlist[0] == 'pool1':
+                struuid = self.container1.get_uuid_str()
+                container_uuid = uuid.UUID(struuid)
+            else:
+                if uuidlist[0] == 'MFUUID':
+                    container_uuid = "misformed-uuid-0000"
```

Comment:
(pylint-redefined-variable-type) Redefinition of container_uuid type from uuid.UUID to str